geomesa-convert/README.md

```diff
@@ -824,5 +827,29 @@ updated file.
             { name = "geom", transform = "point($lon, $lat)" }
           ]
         }
-      ]
+      }
     }
+
+## Loading Converters and SFTs at Runtime
+
+If you have defined converters or SFTs in typesafe config you can place them on the classpath or load them with a 
+ConverterConfigProvider or SimpleFeatureTypeProvider via Java SPI loading. By default, classpath and URL providers are 
+provided. Placing a typesafe config file named ``reference.conf`` containing properly formatted converters and SFTs 
```

Comment:
i think the preferred scheme is for only libraries to use reference.conf, and applications to use application.conf.